xonsh/jobs.py

```diff
@@ -5,25 +5,27 @@
 import time
 import signal
 import builtins
-from subprocess import TimeoutExpired
+from subprocess import TimeoutExpired, check_output
 from io import BytesIO
+from collections import deque
 
 from xonsh.tools import ON_WINDOWS
 
+RUNNING = 1
+STOPPED = 0
```

Comment:
Do we want these as integers, rather than string flags?  And if ints, shouldn't it be an enum?